README.md

```diff
@@ -53,6 +53,23 @@ pip install retworkx
 will build retworkx for your local system from the source package and install
 it just as it would if there was a prebuilt binary available.
 
+### Optional dependencies
+
+If you're planning to use the `retworkx.visualization` module you will need to
+install optional dependencies to use the functions. The matplotlib drawer
+required that the [matplotlib](https://matplotlib.org/) library is installed.
+This can be installed with `pip install matplotlib` or when you're installing
+retworkx with `pip install 'retworkx[mpl]'`. If you're going to use the graphviz
+based drawer first you will need to install graphviz, instructions for this can
+be found here: https://graphviz.org/download/#executable-packages. Then you
+will need to install the [pydot](https://pypi.org/project/pydot/) and
+[pillow](https://python-pillow.org/) Python libraries. This can be done either
+with `pip install pydot pillow` or when installing retworkx with
+`pip install 'retworkx[pydot]'`.
```

Comment:
This might be better phrased as something along the lines of: "Alternatively, you can use graphviz for drawing functionality.  Instructions for installing it can be found [here]...

Reply:
It's less of an alternative and more just a different visualization function that uses a different backend. I've updated the wording to make this a bit clearer in: https://github.com/Qiskit/retworkx/pull/339/commits/500e8923227d787ee51dd1e65c71a48e737dd77d